src/Microsoft.AspNetCore.Mvc.Razor/Internal/DefaultRazorPageFactoryProvider.cs

```diff
@@ -74,18 +74,19 @@ public RazorPageFactoryResult CreateFactory(string relativePath)
         /// <summary>
         /// Creates a factory for <see cref="IRazorPage"/>.
         /// </summary>
-        /// <param name="compiledType">The <see cref="Type"/> to produce an instance of <see cref="IRazorPage"/>
+        /// <param name="pageCreator">The <see cref="Func{Object}"/> to produce an instance of <see cref="IRazorPage"/>
```

Comment:
Since we know the return type and are going to cast to it, why not `Func<IRazorPage>`?